source/routing/asynchronous-routing.md

```diff
@@ -94,19 +94,19 @@ App.TardyRoute = Ember.Route.extend({
 });
 ```
 
-When transitioning into `TardyRoute`, the `model` hook will be called and
+When transitioning into `route:tardy`, the `model` hook will be called and
```

Comment:
I think this is what Steve was asking about below. I agree that the factory name is a little in-the-weeds for the guides, but it might be useful to have something to differentiate between tardy the route and tardy the component. I think that's exactly the kind of distinction that beginners get stuck on. What if it were "When transitioning into tardy (app/routes/tardy)", or some kind of hover, rollover, or annotation to the same effect?